x-pack/plugin/esql/qa/testFixtures/src/main/resources/string.csv-spec

```diff
@@ -1168,3 +1168,48 @@ from employees | where emp_no == 10001 | eval split = split("fooMbar", gender) |
 gender:keyword | split:keyword 
 M              | [foo, bar]
 ;
+
+locate#[skip:-8.13.99,reason:new string function added in 8.14]
+row a = "hello" | eval a_ll = locate(a, "ll");
+
+a:keyword | a_ll:integer
+hello | 2
+;
+
+locateFail#[skip:-8.13.99,reason:new string function added in 8.14]
+row a = "hello" | eval a_ll = locate(a, "int");
+
+a:keyword | a_ll:integer
+hello | -1
+;
+
+locateLongerSubstr#[skip:-8.13.99,reason:new string function added in 8.14]
+row a = "hello" | eval a_ll = locate(a, "farewell");
+
+a:keyword | a_ll:integer
+hello | -1
+;
+
+locateSame#[skip:-8.13.99,reason:new string function added in 8.14]
+row a = "hello" | eval a_ll = locate(a, "hello");
+
+a:keyword | a_ll:integer
+hello | 0
+;
+
+locateWithSubstring#[skip:-8.13.99,reason:new string function added in 8.14]
+from employees | where emp_no <= 10010 | eval f_s = substring(last_name, 2) | eval f_l = locate(last_name, f_s) | keep emp_no, last_name, f_l;
+ignoreOrder:true
+
+emp_no:integer | last_name:keyword | f_l:integer
+10001 | Facello   | 1
+10002 | Simmel    | 1
+10003 | Bamford   | 1
+10004 | Koblick   | 1
+10005 | Maliniak  | 1
+10006 | Preusig   | 1
+10007 | Zielinski | 1
+10008 | Kalloufi  | 1
+10009 | Peac      | 1
+10010 | Piveteau  | 1
+;
```

Comment:
It's probably worth adding one or two examples with a unicode string.

Reply:
good idea, thanks !